physical_property/experimental_data/README.md

```diff
@@ -0,0 +1,11 @@
+# Experimental values of the SAMPL7 challenge molecules
+
+After the SAMPL7 challenge submission deadline, experimental values for the 22 challenge molecules will be made available in this directory.
+
+
+## Manifest
+
+[`experimental_ID_and_SAMPL7_molecule_ID_table.csv`](experimental_ID_and_SAMPL7_molecule_ID_table.csv): A table that matches experimental molecule IDs to SAMPL7 challenge molecule IDs.
+
+## Publication on experimental measurements
+Karol R. Francisco, Thomas J. Paniak, Marisa C. Kozlowski, and Carlo Ballatore. "Structure Property Relationships of N-Acylsulfonamides and Related Bioisosteres" *(Pending Publication)*
```

Comment:
I'd say something like, "Details to be announced," not "Pending publication". The latter makes it sound like it's already accepted.

Reply:
Updated